modin/core/execution/dask/implementations/pandas_on_dask/partitioning/virtual_partition.py

```diff
@@ -418,14 +418,12 @@ def drain_call_queue(self, num_splits=None):
         num_splits : int, default: None
             The number of times to split the result object.
         """
-
-        def drain(df):
-            for func, args, kwargs in self.call_queue:
-                df = func(df, *args, **kwargs)
-            return df
-
+        # TODO: Need to check if `drain_call_queue` speeds up if helper
```

Comment:
Should we check this before merging the code?

Reply:
I think it can be done separately.

Reply:
When do you want to remove this comment?

Reply:
When someone who profiles on Dask sees the problem or lack of it.